modules/nodes-nodes-problem-detector-installing.adoc

```diff
@@ -3,33 +3,46 @@
 // * nodes/nodes-nodes-problem-detector.adoc
 
 [id='nodes-nodes-problem-detector-installing_{context}']
-= Installing the {product-title} node problem detector
+= Installing the {product-title} Node Problem Detector
 
-You can use the {product-title} console to subscribe to the Node Problem Detector subscription, 
-which creates the Node Problem Detector Operator.
+You can use the {product-title} console to install the Node Problem Detector (NPD), which creates the Node Problem Detector Operator.
 
-.Procedure
+.Prerequisites
 
-To install the Node Problem Detector:
+. Create a Project for the NPD:
++
+[source,bash]
+----
+$ oc create ns openshift-node-problem-detector
+----
 
-.Procedure
+. Create an Operator Group
++
+[source,bash]
+----
+$ oc create -f - <<EOF
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: npd-operators
+  namespace: openshift-node-problem-detector
+spec:
+  targetNamespaces:
+  - openshift-node-problem-detector
+EOF
+----
 
-To subscribe to the subscription:
+.Procedure
 
-. In the {product-title} console, click *Catalog* -> *Operator Hub* -> *node-problem-detector*.
+To install the Node Problem Detector:
 
-. In the *node-problem-detector* overview, click *Install*.
+. In the {product-title} console, click *Catalog* -> *Operator Hub*. 
 
-. In *Create Operator Subscription*: 
+. Choose  *node-problem-detector* from the list of available Operators, and click Install.
 
-.. Select the operator.
+. On the *Create Operator Subscription* page change the *Target* to the *global-operators* Operator Group. This makes the Operator available to all users and projects that use this OpenShift Container Platform cluster.
```

Comment:
@joelsmith @sjenning should we charge the *Target* to OG npd-operators? Since we just got it created on previous step.  If using OG global-operators, then why should we create OG npd-operators?